cfgov/cfgov/urls.py

```diff
@@ -250,6 +250,7 @@
     urlpatterns.append(pattern)
 
 if settings.DEBUG :
+    urlpatterns.append(url(r'^test-fixture/$', SheerTemplateView.as_view(template_name='test-fixture/index.html'), name='test-fixture'))
```

Comment:
Doesn't look like bewd's are spacing it on the python-side. It'd be hundreds of edits for the parens in this file. Prob best for another PR.